spam/01_spam_tutorial.py

```diff
@@ -3,17 +3,31 @@
 # # 🚀 Snorkel Intro Tutorial: Spam Detection
 
 # %% [markdown]
-# In this tutorial, we will walk through the process of using Snorkel to build a training set for classifying YouTube comments as `SPAM` or `HAM` (not spam).
+# In this tutorial, we will walk through the process of using Snorkel to build a training set for classifying YouTube comments as spam or not spam.
 # The goal of this tutorial is to illustrate the basic components and concepts of Snorkel in a simple way, but also to dive into the actual process of iteratively developing real applications in Snorkel.
 # For an overview of Snorkel, visit [snorkel.org](http://snorkel.org).
 # You can also check out the [Snorkel API documentation](https://snorkel.readthedocs.io/).
 #
 # For our task, we have access to a large amount of *unlabeled data* in the form of YouTube comments with some metadata.
-# Our goal is to train a classifier over the comment data that can predict whether a comment is spam or not.
-# To do that, we need to label our data, but doing so for real world applications can be prohibitively slow and expensive, often taking person-weeks or months of hand-labeling.
-#
-# We therefore turn to weak supervision using **_labeling functions_**, or noisy, programmatic heuristics, to assign labels to unlabeled training data efficiently.
-# We also have access to a small amount of labeled data, which we only use for evaluation purposes.
+# Our goal is to train a classifier over the comment data that can predict whether a comment is spam or not spam.
```

Comment:
switch goal of building a classifier and having access to unlabeled data sentences

Reply:
wow my internal parser needed a few passes on that one. but 👍